internal/cfgplugins/policyforwarding.go

```diff
@@ -65,6 +65,11 @@ type OcPolicyForwardingParams struct {
 	PolicyName         string              // PolicyName refers to the traffic policy that is bound to the given interface in CLI-based configuration.
 	NetworkInstanceObj *oc.NetworkInstance // NetworkInstanceObj represents the OpenConfig network instance (default/non-default VRF).
 	HasMPLS            bool                // HasMPLS indicates whether the policy forwarding configuration involves an MPLS overlay.
+	ActionSetTTL       bool
+	MatchTTL           int
+	RwTTL              int
+	NHGName            string
```

Comment:
To establish a consistent pattern for policy-forwarding actions rename this parameter to:

`ActionNHGName`